ci/azure-jobs-linux.yml

```diff
@@ -30,7 +33,13 @@ jobs:
       # Pass modified PATH to following steps:
       echo "##vso[task.setvariable variable=PATH]$PATH"
     displayName: Install Dependencies
-  - bash: ./ci/build_librepcb.sh
+  - bash: rm -rf libs/quazip
+    displayName: Remove unbundled quazip
+    condition: and(succeeded(), contains(variables['UNBUNDLE'], 'quazip'))
+  - bash: |
+      ./ci/build_librepcb.sh
+      # Set LD_LIBRARY_PATH
+      echo "##vso[task.setvariable variable=LD_LIBRARY_PATH]$(pwd)/build/install/opt/lib"
```

Comment:
Why not simply set the `LD_LIBRARY_PATH` right in the build matrix above? Then it would even be set only on the build job with unbundled libraries...

Reply:
Because it needs to be an absolute path, and Azure doesn't show the full directory paths... I don't think we can rely on the checkout directory to remain a constant, right?

Reply:
Sure, but you can set it right in the build matrix like that:

```
LD_LIBRARY_PATH: $(Agent.BuildDirectory)/build/install/opt/lib
```

Reply:
Ah, I wasn't aware of those variables! Will update the config.